Messages.cs

```diff
@@ -28,6 +28,15 @@ public static string
               EnterPhoneNumber = string.Format("Enter Phone Number:{0}", Environment.NewLine),
               InvalidOptionSelected = string.Format("Invalid Option entered, try again:{0}", Environment.NewLine),
               InvalidNumberEntered = string.Format("Invalid Number entered, try again:{0}", Environment.NewLine),
-              GarageMenu = string.Format("Hello to our Garage:{0}1.Add New Car{0}2.See all license Number Vehicle in garage{0}3.Change status of car{0}4.fill air pressure in wheels of vehicle to Maximum{0}5.fill fuel vehicle{0}6.fill electric vehicle{0}7.show all details of vehicle", Environment.NewLine);
+              GarageMenu = string.Format(@"Hello to our Garage:{0}
```

Comment:
No need Environment.NewLine if you are using @
because it includes the New Line in the text already